Tests/MapboxNavigationTests/ImageLoadingURLProtocolSpy.swift

```diff
@@ -72,16 +72,13 @@ final class ImageLoadingURLProtocolSpy: URLProtocol {
 
                 switch result {
                 case .success(let imageData):
-                    guard let image: UIImage = UIImage(data: imageData) else {
-                        XCTFail("Invalid image data found"); return
-                    }
                     // send an NSHTTPURLResponse to the client
```

Comment:
To remove the check of the `UIImage(data: imageData)` is because the `SpriteRepository` is using the `ImageDownloader` to download the data for `SpriteMetaDataCache` as well. The data is not for UIImage, and it blocks the test of `testDownLoadingSpriteMetaData`.